src/Microsoft.AspNetCore.SignalR.Client/HubConnectionBuilderHttpExtensions.cs

```diff
@@ -8,50 +8,109 @@
 
 namespace Microsoft.AspNetCore.SignalR.Client
 {
+    /// <summary>
+    /// Extensions methods for <see cref="IHubConnectionBuilder"/>.
+    /// </summary>
     public static class HubConnectionBuilderHttpExtensions
     {
+        /// <summary>
+        /// Adds a URL for configuring a <see cref="HttpConnection"/> the <see cref="HubConnection"/> will use.
+        /// </summary>
+        /// <param name="hubConnectionBuilder">The <see cref="IHubConnectionBuilder" /> to configure.</param>
+        /// <param name="url">The URL the <see cref="HttpConnection"/> will use.</param>
+        /// <returns>The same instance of the <see cref="IHubConnectionBuilder"/> for chaining.</returns>
         public static IHubConnectionBuilder WithUrl(this IHubConnectionBuilder hubConnectionBuilder, string url)
         {
             hubConnectionBuilder.WithUrlCore(new Uri(url), null, _ => { });
             return hubConnectionBuilder;
         }
 
+        /// <summary>
+        /// Adds a URL and delegate for configuring a <see cref="HttpConnection"/> the <see cref="HubConnection"/> will use.
+        /// </summary>
+        /// <param name="hubConnectionBuilder">The <see cref="IHubConnectionBuilder" /> to configure.</param>
+        /// <param name="url">The URL the <see cref="HttpConnection"/> will use.</param>
+        /// <param name="configureHttpConnection">The delegate that configures the <see cref="HttpConnection"/>.</param>
+        /// <returns>The same instance of the <see cref="IHubConnectionBuilder"/> for chaining.</returns>
         public static IHubConnectionBuilder WithUrl(this IHubConnectionBuilder hubConnectionBuilder, string url, Action<HttpConnectionOptions> configureHttpConnection)
         {
             hubConnectionBuilder.WithUrlCore(new Uri(url), null, configureHttpConnection);
             return hubConnectionBuilder;
         }
 
+        /// <summary>
+        /// Adds a URL and transport for configuring a <see cref="HttpConnection"/> the <see cref="HubConnection"/> will use.
+        /// </summary>
+        /// <param name="hubConnectionBuilder">The <see cref="IHubConnectionBuilder" /> to configure.</param>
+        /// <param name="url">The URL the <see cref="HttpConnection"/> will use.</param>
+        /// <param name="transports">The transport the <see cref="HttpConnection"/> will use.</param>
```

Comment:
`A bitwise-OR of fields from the [HttpTransportType] flags enum indicating which transports the client should use`